pom.xml

```diff
@@ -1095,6 +1095,7 @@
 
             <!-- Used in internal utilities (protected by conditional) -->
             <ignore>java.lang.Long</ignore>
+            <ignore>java.lang.Character</ignore>
```

Comment:
doesn't look like it ... i guess I'll just remove this (and the other usage for Long.compare ... which also doesn't look like it uses intrinsics)